src/installation/guides/chroot.md

```diff
@@ -0,0 +1,252 @@
+
+These instructions can be followed on any running Linux system that has xbps
+installed, including an already working installation of Void or an official
+[live image](../live-images/prep.md). It is assumed that the host computer is
+connected to the Internet.
+
+[Partition your disks](../live-images/partitions.md) and format them using
+[mke2fs(8)](https://man.voidlinux.org/mke2fs.8),
+[mkfs.xfs(8)](https://man.voidlinux.org/mkfs.xfs.8),
+[mkfs.btrfs(8)](https://man.voidlinux.org/mkfs.btrfs.8) or whatever tools are
+necessary for your filesystem(s) of choice.
+
+[cfdisk(8)](https://man.voidlinux.org/cfdisk.8) and
+[fdisk(8)](https://man.voidlinux.org/fdisk.8) are available on live
+distributions for partitioning, but you may wish to use
+[gdisk(8)](https://man.voidlinux.org/gdisk.8) (from the package `gptfdisk`) or
+[parted(8)](https://man.voidlinux.org/parted.8) instead.
+
+NB For a UEFI booting system, make sure to create a UEFI System Partition (ESP).
+The ESP should have the partition type "EFI System" (code ef00) and be formatted
+as fat32 using [mkfs.vfat(8)](https://man.voidlinux.org/mkfs.vfat.8).
+
+### Create a New Root and Mount Filesystems
+
+Create an empty directory to serve as a mount point for the new root filesystem,
+then mount any other filesystems on top of this new root. `/mnt` may be used
+instead of creating a new directory.
+
+This directory will be called `$VDROOT` for the remainder of this guide.
+
+```
+# VDROOT=/mnt
+```
+
+If using UEFI, mount the EFI System Partition as `$VDROOT/boot/efi`.
+
+For example, if `/dev/sda2` is to be mounted as `/` and `/dev/sda3` is to be
+booted as `/home` and `dev/sda1` is the EFI System Partition
+
+```
+# mkdir -p "$VDROOT"
+# mount /dev/sda2 "$VDROOT"/
+# mkdir "$VDROOT"/boot/
+# mount /dev/sda3 "$VDROOT"/home/
+# mkdir -p "$VDROOT"/boot/efi
+# mount /dev/sda1 "$VDROOT"/boot/efi
+```
+
+Initiliazile swap space, if desired using
+[mkswap(8)](https://man.voidlinux.org/mkswap.8).
+
+## Base Installation
+
+[Select a mirror to download packages
+from.](../../xbps/repositories/mirror/index.md) For simplicity, save this URL to
+a shell variable for later use. Append `/current` to this repository url to
+install a glibc distribution, append `/current/musl` for a musl distribution (NB
+musl binaries are not available for i686).
+
+For example,
+
+```
+# REPO=https://alpha.de.repo.voidlinux.org/current
+```
+
+xbps also needs to know what architecture is being installed, available options
+are `x86_64`, `x86_64-musl` and `i686` for PC architecture computers.
+
+For example,
+
+```
+# ARCH=x86_64
+```
+
+This architecture must be compatible with your current operating system, but
+does not need to be the same. If your host is running an x86_64 operating
+system, any of the three architectures can be installed (whether the host is
+musl or glibc), but an i686 host can only install i686 distributions.
+
+Three metapackages are available to boot strap Void: `base-system`,
```

Comment:
I think voidstrap is specifically intended for container or chroot environments (not intended to become system installs.) I'm not exactly sure the distictions between minimal and system beyond the lack of a kernel.

Reply:
base minimal is lacking *a lot* of packages that base-system has. I usually use it for chroot installs but after looking through the package dependencies I've realized that I end up installing pretty much all of base-system anyway. base-minimal is probably only useful in very limited situations. I'll just take out the whole discussion.

Reply:
Yeah, I think base-minimal is useful when you *know* what packages you need and you want a different kernel. It would be good to document somewhere else :)

Reply:
It was mainly added for smaller docker containers at the time.

Base minimal is really not that useful for baremetal since we also have `ignorepkg=`, its easier to ignore one or two packages from base-system than installing all the extra packages required to make `base-minimal` boot on hardware, which roughly would be `linuxX.X`, `eudev`, `iproute2`, `ncurses`, `kbd`, (filesystem stuff like `e2fsprogs`), `kmod` and then you are still missing some more important but optional packages like `less`, `mdocml` and I'm probably forgetting a few required and important optional ones.